deps/rabbit/src/rabbit_quorum_queue.erl

```diff
@@ -406,7 +406,14 @@ handle_tick(QName,
                                {messages_persistent, M}
 
                                | infos(QName, ?STATISTICS_KEYS -- [consumers])],
-                      rabbit_core_metrics:queue_stats(QName, Infos),
+                      {ok, Queue} = rabbit_amqqueue:lookup(QName),
```

Comment:
This does not handle the case where the queue record no longer exists.

Reply:
If the queue record doesn't exist anymore, will the `handle_tick` function still be called for the queue?
I thought it's safe to pattern match here give the same is done in https://github.com/rabbitmq/rabbitmq-server/blob/4465071820e143d8f380baca9dd2b4802da42840/deps/rabbit/src/rabbit_quorum_queue.erl#L425
? However, I might be wrong.

Reply:
It can be deleted concurrently. I guess then the process will typically be terminating, too, so this is good enough.

Reply:
you shouldn't need to get the queue record at all as all the information you need should be passed to this function from the tick.